source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -140,6 +238,117 @@ TEST_F(NiDCPowerDriverApiTest, NiDCPowerReset_SendRequest_ResetCompletesSuccessf
   expect_api_success(response.status());
 }
 
+TEST_F(NiDCPowerDriverApiTest, NiDCPowerSetViInt32Attribute_SendRequest_GetViInt32AttributeMatches)
```

Comment:
- The test class is already named `NiDCPowerDriverApiTest`. The test names shouldn't need the `NiDCPower` prefix too.
- The action step on all these methods isn't a call to `SendRequest`. Consider making the call to `SetAttributeViInt32` the action and just not declaring a "where" clause: `SetAttributeViInt32_GetAttributeViInt32ReturnsSameValue`.

Reply:
I have removed the NiDCPower prefix from all the tests and changed the name of the tests as per the suggestion.

Reply:
Done.